Makefile

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 DOCKER_TAG ?= 1.4.0
 SIDECAR_TAG ?= 1.4.0
+VERSION_TAG ?= 1.4-0
```

Comment:
VERSION_TAG is 1.4-0 and DOCKER_TAG is 1.4.0?

Reply:
Is this comment related to the other comment thread?